WordPress/Classes/ViewRelated/Stats/Charts/StatsBarChartView.swift

```diff
@@ -135,6 +178,19 @@ class StatsBarChartView: BarChartView {
         applyStyling()
         configureAndPopulateData()
     }
+
+    private func redrawChartMarkersIfNeeded() {
+        guard marker != nil, let highlight = lastHighlighted, let entry = barData?.entryForHighlight(highlight) else {
+            return
+        }
+
+        notifyDataSetChanged()
+
+        let postRotationDelay = DispatchTime.now() + TimeInterval(0.3)
```

Comment:
Is this just required so that it happens after the rotation is complete?

Reply:
`notifyDataSetChanged` redraws the chart post-rotation, but the framework does not necessarily understand how the custom marker view should be redrawn in response.

Without a delay, I observed that the call from `getBarBounds(entry:)` was incorrect, and it wasn't rendering correctly.